include/openPMD/Mesh.hpp

```diff
@@ -69,15 +70,27 @@ class Mesh : public BaseRecord< MeshRecordComponent >
     };  //DataOrder
 
     /**
-     * @return String representing the geometry of the mesh of the mesh record.
+     * @return Enum representing the geometry of the mesh of the mesh record.
      */
     Geometry geometry() const;
+    /**
+     * @return String representing the geometry of the mesh of the mesh record.
+     */
+    std::string geometryString() const;
     /** Set the geometry of the mesh of the mesh record.
      *
      * @param   g    geometry of the mesh of the mesh record.
      * @return  Reference to modified mesh.
      */
     Mesh& setGeometry(Geometry g);
+    /** Set the geometry of the mesh of the mesh record.
+     * If the geometry is unknown to the openPMD-api, the string is prefixed
```

Comment:
Doxygen title/body is separated by a newline:
```suggestion
     *
     * If the geometry is unknown to the openPMD-api, the string is prefixed
```